server/services/index.ts

```diff
@@ -49,6 +50,7 @@ export const services = () => {
     enabled: config.apis.audit.enabled,
   })
 
+  const personalLearningPlansService = PersonalLearningPlanServiceFactory.getInstance(dataAccess)
```

Comment:
We are using a factory to get the right instance of the service.
I think this is the right approach. Currently there are 2 possible service implementations - 1 to get PLP data from Education & Work Plan API (for use where the feature toggle is enabled), and a noop implementation for use where the feature toggle is disabled (meaning we dont need too much logic re: feature toggles too deep into the stack 👍 )
Longer term the plan is to replace the service implementation that uses the Education & Work Plan API with one that gets data from the OnePlan API, so this use of a factory will likely pay dividends then as well